package/MDAnalysis/analysis/align.py

```diff
@@ -359,27 +415,141 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
+        start : int, optional
+            First frame of trajectory to analyse, Default: 0
+        stop : int, optional
+            Last frame of trajectory to analyse, Default: -1
+        step : int, optional
+            Step between frames to analyse, Default: 1
+
+        """
+        self.quiet = quiet
+        if self.quiet:
+            # should be part of a try ... finally to guarantee restoring the log level
+            logging.disable(logging.WARN)
+
+        self.mobile = mobile
+        self.reference = reference
+        frames = self.mobile.trajectory
+
+        select = rms.process_selection(select)
+        self.ref_atoms = self.reference.select_atoms(*select['reference'])
+        self.mobile_atoms = self.mobile.select_atoms(*select['mobile'])
+        self.natoms = self.mobile_atoms.n_atoms
+
+        kwargs.setdefault('remarks', 'RMS fitted trajectory to reference')
+
+        if filename is None:
+            path, fn = os.path.split(frames.filename)
+            filename = os.path.join(path, prefix + fn)
+            logger.info('none filename: {0}'.format(filename))
+
+        if os.path.exists(filename) and not force:
+            raise IOError('Filename already exists in path and force is not set to True')
+        self.filename = filename
+
+        self.natoms = self.mobile_atoms.n_atoms
+        self.ref_atoms, self.mobile_atoms = get_matching_atoms(self.ref_atoms,
+                            self.mobile_atoms, tol_mass=tol_mass, strict=strict)
+
+        self.writer = mda.Writer(filename, self.natoms)
+
+        if mass_weighted:
+            # if performing a mass-weighted alignment/rmsd calculation
+            self.weights = self.ref_atoms.masses
+        else:
+            self.weights = None
+
+        logger.info("RMS-fitting on {0:d} atoms.".format(len(self.ref_atoms)))
+        self._setup_frames(frames, start,stop,step)
+
+    def _prepare(self):
+        # reference centre of mass system
+        self.ref_com = self.ref_atoms.center_of_mass()
+        self.ref_coordinates = self.ref_atoms.positions - self.ref_com
+        # allocate the array for selection atom coords
+        self.rmsd = np.zeros((self.nframes,))
+
+    def _single_frame(self):
+        index = self._ts.frame
+
+        self.mobile_com = self.mobile_atoms.center_of_mass()
+        self.mobile_coordinates = self.mobile_atoms.positions.copy() - self.mobile_com
+
+        self.mobile_atoms, self.rmsd[index] = _fit_to(self.mobile_coordinates,\
+            self.ref_coordinates, self.mobile_atoms, self.mobile_com,\
+            self.ref_com, self.weights)
+         # write whole aligned input trajectory system
+        self.writer.write(self.mobile_atoms)
+
+    def _conclude(self):
+        self.writer.close()
+        if self.quiet:
+            logging.disable(logging.NOTSET)
+
+    def save(self, rmsdfile):
+        np.savetxt(rmsdfile, self.rmsd)
+        logger.info("Wrote RMSD timeseries  to file %r", rmsdfile)
```

Comment:
What RMSD series? Is this the RMSD after an alignment. Did I only calculate the RMSD? Please give some minimal context in the logging messages so that one can later understand what has been done when actually looking at the logs.